src/System Application/App/Extension Management/src/ExtensionInstallation.Table.al

```diff
@@ -33,5 +33,12 @@ table 2503 "Extension Installation"
         {
             Caption = 'Preview Key';
         }
+        /// <summary>
+        /// The telemetry URL for the Partner Center ingestion call back to be used when installing a version of an AppSource app.
+        /// </summary>
+        field(3; RESPONSEURL; Text[2048])
```

Comment:
I will double check because it has been some time since I added PREVIEWKEY, but AFAIR the table field's casing has to be matching the one in URL filter for the installation link on page 2503. Since it was using ID all upper case, I used PREVIEWKEY too. For the response URL, this isn't surfaced in the install link (at least for now), but I made it upper case for consistency. I agree, it looks a bit weird (hence my comment on the table header), but in AL, it doesn't really matter since it's not case sensitive so you can reference it with PascalCase in code. 

Regarding the caption, I can update it - not sure why I made it inconsistent.

Reply:
Caption was updated. I also verified that the casing must match the one of the table. Waiting on decision regarding the casing, but likely going to stay this way.

Reply:
Discussed. Casing changed to PreviewKey!